folsom/src/main/java/com/spotify/folsom/MemcacheClientBuilder.java

```diff
@@ -399,14 +399,27 @@ private void updateResolver() {
   }
 
   /**
-   * Enforce a timeout for requests to complete, closing the connection and reconnecting if the
-   * timeout is exceeded.
+   * This has been deprecated - see {#link withConnectionTimeoutMillis}.
    *
    * @param timeoutMillis The timeout in milliseconds. The default is 3000 ms.
    * @return itself
    */
+  @Deprecated
   public MemcacheClientBuilder<V> withRequestTimeoutMillis(final long timeoutMillis) {
-    this.timeoutMillis = timeoutMillis;
+    return withConnectionTimeoutMillis(timeoutMillis);
+  }
+
+  /**
+   * Set the maximum time to wait before considering the connection to be dead and should be
+   * closed and recreated.
+   *
+   * Do not use this to enforce request timeouts
```

Comment:
Should we give a clue on what to do instead?

Reply:
Added to readme - add something here too?